ocpmodels/trainers/energy_trainer.py

```diff
@@ -253,7 +255,7 @@ def train(self):
                         split="train",
                     )
 
-            self.scheduler.step()
+                self.scheduler.step()
```

Comment:
The energy trainer currently only saves checkpoints at each epoch - did we want to add that on a per-step basis like we did with `forces_trainer`: https://github.com/Open-Catalyst-Project/ocp/pull/177/files#diff-b05769e2764b8b45457d8c3f0b7b203db2c100a8ab569849ae5174fc1a71f53dR413.

Given that IS2RE are fairly quick to train I can see us passing on this, good to make sure though.

Reply:
Yeah, it's less important because IS2RE is really fast.